packages/tooltips/README.md

```diff
@@ -39,7 +39,7 @@ their parent [ThemeProvider](https://zendeskgarden.github.io/react-components/th
 
 #### English (LTR) Placements
```

Comment:
The benefit of these LTR/RTL sections are questionable (and not easily digestible). All Garden components respond to RTL. The callout here feels a bit gratuitous and could probably just be removed.

Reply:
Agreed, just wasn't sure if we wanted to remove that much content. I've removed both sections!